contracts/scripts/defender-actions/harvest.js

```diff
@@ -32,25 +27,17 @@ const handler = async (event) => {
     ethersVersion: "v5",
   });
 
-  const network = await provider.getNetwork();
-  const networkName = network.chainId === 1 ? "mainnet" : "holesky";
-  log(`Network: ${networkName} with chain id (${network.chainId})`);
-
-  const harvesterAddress = addresses[networkName].OETHHarvesterSimpleProxy;
+  const harvesterAddress = addresses.mainnet.OETHHarvesterSimpleProxy;
   log(`Resolved OETH Harvester Simple address to ${harvesterAddress}`);
   const harvester = new ethers.Contract(harvesterAddress, harvesterAbi, signer);
 
-  const convexAMOProxyAddress = addresses[networkName].ConvexOETHAMOStrategy;
-
-  // Always harvest from Convex AMO
-  const strategiesToHarvest = [convexAMOProxyAddress];
-
   const nativeStakingStrategies = [
     // addresses[networkName].NativeStakingSSVStrategyProxy,
-    addresses[networkName].NativeStakingSSVStrategy2Proxy,
-    addresses[networkName].NativeStakingSSVStrategy3Proxy,
+    addresses.mainnet.NativeStakingSSVStrategy2Proxy,
+    addresses.mainnet.NativeStakingSSVStrategy3Proxy,
```

Comment:
nit: Should we add a `chainId == 1` check at the start of the script and revert if it's not the case? I don't think it's necessary since the underlying txs will fail with addresses on wrong network either way. But it might be easier to debug in such a case

Reply:
Good idea. I've fixed in commit https://github.com/OriginProtocol/origin-dollar/pull/2725/commits/e85418f63598f0970396b45a06816f11c5e0245e